### DISCUSSION TITLE
Can I highlight messages from users with certain badges in /mention

### DISCUSSION THREAD

**Discussion by 17Comic** [Chatterino/chatterino2#3323] [Ideas]
I would like to show only chats from users that are VIPs and Moderators, and I can highlight their messages in the streamers chat but I can't press show in /mention and exclude all other chats that I haven't chosen to highlight.

**Comment by Felanbird** [comment]
You can simulate this with filters, but if you do this for mentions via `badge highlights`, it's likely to still cause other annoyances.

You can apply a filter to your /mentions tab, possibly looking something like this:
`(channel.name == "pajlada" && (author.badges contains "vip" || author.badges contains "moderator")) || message.content contains "17comic"` but taking this approach will still highlight all of the other tabs when mods/vips speak.

If you wanted to not utilize your mentions, you could apply the filter to a duplicate of the channel itself.
`(channel.name == "pajlada" && (author.badges contains "vip" || author.badges contains "moderator"))`

1. Normal Chat
2. Filtering only mods and vips for the channel
3. Filtering mods and vips who spoke in `pajlada` for `/mentions` 
- (I actually highlighted only supibot in example 3, but the badge highlights will work the same way)
![image](https://user-images.githubusercontent.com/41973452/139524778-cd8ec5a9-a8cd-44d3-8bef-7a3edede9476.png)

**Reply by 17Comic** [reply]
thanks for the help! im just confused on where to put in the code for the filters, is there a place for me to type it in?

**Reply by Felanbird** [reply]
> 
> 
> thanks for the help! im just confused on where to put in the code for the filters, is there a place for me to type it in?

⚙️ -> Filters -> Add -> ok | and then just copy paste into the new box you've added.
To toggle it on, right click the channel name [or /mentions] at the top of the split and click `Set filters`
![image](https://user-images.githubusercontent.com/41973452/139561996-efefe5d5-138a-43d8-8724-20e10f521069.png)
